cobigen/cobigen-core-api/src/main/java/com/devonfw/cobigen/api/util/to/SearchResponseUtil.java

```diff
@@ -0,0 +1,97 @@
+package com.devonfw.cobigen.api.util.to;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import org.glassfish.jersey.client.oauth2.OAuth2ClientSupport;
+
+import com.devonfw.cobigen.api.exception.RESTSearchResponseException;
+
+import jakarta.ws.rs.client.Client;
+import jakarta.ws.rs.client.ClientBuilder;
+import jakarta.ws.rs.client.Invocation;
+import jakarta.ws.rs.client.WebTarget;
+import jakarta.ws.rs.core.Feature;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+
+/**
+ *
+ * Utility class for search REST API requests handling
```

Comment:
Okay, that's more of a general question.
Why are we using Utility classes over service classes? Utility classes are undermining the concept of object oriented programming and adding more technical debt than service classes witch is even more of an issue in a project like that where we have lots of fluctuation of developers.

Reply:
Nothing to adjust as discussed.